### DISCUSSION TITLE
How to map `dd` for deletion ?

### DISCUSSION THREAD

**Discussion by ghost** [gokcehan/lf#981] [Q&A] [answered]
Want to imitate vim. I know I can map it directly to some other key. But is there any way to map it to 'dd' (as in double press) ?

**Comment by ilyagr** [comment, marked as answer]
Yes. `:map dd <something>` should just work.

**Reply by ghost** [reply]
Tried. 1st press is move selection asusual. 2nd press doesn't do anything.

**Reply by xezwary** [reply]
unmap d first `:map d`

**Reply by ghost** [reply]
Oh I haven't thought of that. Amazing. Works great now.
I had to move cut to `dc` & I've got a sublist when I press `d` 😁
